SEO excerpts: Use part 3 (#3832)
timescale/docs: use-timescale/data-tiering/about-data-tiering.md

---
title: About the object storage tier

> excerpt: LLearn how Timescale Cloud helps you save on storage costs with its tiered storage architecture that includes a high-performance storage tier and a low-cost object storage tier built on Amazon s3

Reviewer: ```suggestion
excerpt: Learn how Timescale Cloud helps you save on storage costs. The Timescale Cloud tiered storage architecture includes a high-performance storage tier and a low-cost object storage tier built on Amazon s3
```
